app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ProjectsController < ApplicationController
+  def show
+    @current_project ||= Project.find_by_slug(params[:id])
```

Comment:
I think we should also figure out how to pass the slug in under params[:slug], I'll make a note to refactor.